rs/nns/governance/src/governance/benches.rs

```diff
@@ -512,3 +513,58 @@ fn compute_ballots_for_new_proposal_with_stable_neurons() -> BenchResult {
             .expect("Failed!");
     })
 }
+
+fn list_neurons_benchmark() -> BenchResult {
+    let neurons = (0..100)
+        .map(|id| {
+            (id, {
+                let mut neuron: NeuronProto = make_neuron(
+                    id,
+                    PrincipalId::new_user_test_id(id),
+                    1_000_000_000,
+                    hashmap! {}, // get the default followees
+                )
+                .into_proto(&VotingPowerEconomics::DEFAULT, 123_456_789);
+                neuron.hot_keys = vec![PrincipalId::new_user_test_id(1)];
+                neuron
+            })
+        })
+        .collect::<BTreeMap<u64, NeuronProto>>();
+
+    let governance_proto = GovernanceProto {
+        neurons,
+        ..GovernanceProto::default()
+    };
+
+    let governance = Governance::new(
+        governance_proto,
+        Box::new(MockEnvironment::new(Default::default(), 0)),
+        Box::new(StubIcpLedger {}),
+        Box::new(StubCMC {}),
+    );
+
+    let request = ListNeurons {
+        neuron_ids: vec![],
+        include_neurons_readable_by_caller: true,
+        include_empty_neurons_readable_by_caller: Some(true),
```

Comment:
Can we make it `Some(false)`? I think that would be more interesting since it's what the NNS Dapp does and what we want to do in the future.

Reply:
done